### PR TITLE
twitter_archiver timeout on sanitize url and added try except 

### PR DESCRIPTION
**Pull request by djhmateer**
A few tweaks.

### CODE REVIEW

File: src/auto_archiver/archivers/twitter_archiver.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ def sanitize_url(self, url: str) -> str:
         # expand URL if t.co and clean tracker GET params
         if 'https://t.co/' in url:
             try:
-                r = requests.get(url)
+                # timeout for urls like https://t.co/KxNwn5yJz6 which caused this request to never come back
+                r = requests.get(url, timeout=30)
```
Comment (msramalho) [inline src/auto_archiver/archivers/twitter_archiver.py:35]: this makes sense

File: src/auto_archiver/archivers/twitter_archiver.py [hunk 2]
```diff
@@ -52,11 +53,21 @@ def download(self, item: Metadata) -> Metadata:
         result = Metadata()
 
         scr = TwitterTweetScraper(tweet_id)
+        do_alternative = False
```
Comment (msramalho) [inline src/auto_archiver/archivers/twitter_archiver.py:56]: What is the goal of this hardcoded value? It will never trigger the condition below.

**Comment by msramalho** [comment]
Thanks for bringing attention to this code.
I think I'll just refactor the entire file logic as it's now doing the attempts in a nested way and not in the best (as in most likely to work) order. I'll include some of the changes you mentioned.

**Comment by msramalho** [comment]
Done, please see https://github.com/bellingcat/auto-archiver/pull/142

closing this PR

**Comment by djhmateer** [comment]
Thanks Miguel

**Pull request closed without merging**